README.md

```diff
@@ -34,6 +34,23 @@ What does it means to design with code? Is it a way of thinking? Is it a tool? I
 
 I don't know where this will take me but there might be 🐉 and 🐛.  
 
+<<<<<<< HEAD
```

Comment:
This is from having and conflict. I think you are working with a not up to date fork of the repo.